catroid/src/main/java/org/catrobat/catroid/ui/ProgramMenuActivity.java

```diff
@@ -82,7 +82,7 @@ public void onCreate(Bundle savedInstanceState) {
 			Sprite sprite = ProjectManager.getInstance().getCurrentSprite();
 			Scene scene = ProjectManager.getInstance().getCurrentScene();
 			if (sprite != null && scene != null && actionBar != null) {
-				String title = scene.getName() + ": " + sprite.getName();
+				String title = ProjectManager.getInstance().getCurrentProject().getSceneList().size() > 1 ? scene.getName() + ": " + sprite.getName() : sprite.getName();
```

Comment:
Internationalization, RTL support. However, thinking about it, it's not that easy after all.

I heard from @wslany that someone is (or was) working on RTL support. What's the progress on that? What does he suggests?